mon-pix/app/components/challenge-item-generic.js

```diff
@@ -49,6 +49,7 @@ export default class ChallengeItemGeneric extends Component {
   @action
   setChallengeAsTimedOut() {
     this.hasChallengeTimedOut = true;
+    this.args.timedoutChallenge();
```

Comment:
si c'est une fonction, on devrait plutôt avoir un verbe d'action
```suggestion
    this.args.timeoutChallenge();
```